plugins/org.eclipse.n4js.ui/plugin.xml

```diff
@@ -1455,4 +1455,48 @@ Contributors:
             class="org.eclipse.n4js.ui.N4JSExecutableExtensionFactory:org.eclipse.n4js.ui.quickfix.packagejson.N4JSPackageJsonQuickfixProviderExtension">
       </quickfixProvider>
    </extension>
+   
+   
+   <!--N4JS Stack Trace Console-->
+   <extension
+         point="org.eclipse.ui.console.consoleFactories">
+      <consoleFactory
+            label="N4JS Stack Trace Console"
+            class="org.eclipse.n4js.ui.N4JSExecutableExtensionFactory:org.eclipse.n4js.ui.console.N4JSStackTraceConsoleFactory">
+      </consoleFactory>
```

Comment:
OK, I see it now. nodejs uses a `ProcessConsole` and we have a custom `N4JSStackTraceConsole` which is created when we open it.

Reply:
Can we somehow use `N4JSStackTraceConsole` instead of `ProcessConsole` as a console for nodejs.

Reply:
These are two different things: The "normal" ProcessConsole is the IO-Console, that is used for all processes. It can have arbitrary output (whatever the application writes to stdout and stderr, also supporting stdin by the way). This console is more ore less hardly wired to the execution of processes from within the Eclipse IDE. I don't think that we should change that, in particular since this console is rather powerful (even if you do not see that on the first glance). So adding a hyperlink feature there is the way to go, and I did.

Reply:
The N4JSStrackTraceConsole is for copying arbitrary content (e.g. from running a node.js process from command line).